src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreSummary.cs

```diff
@@ -26,6 +26,18 @@ public class RestoreSummary
 
         public int InstallCount { get; }
 
+        /// <summary>
+        /// A boolean that specifies if NuGetAudit verified packages for known vulnerabilities
+        /// </summary>
+        /// <remarks>This could be false if NuGetAudit is disabled, if
```

Comment:
```suggestion
        /// <remarks>This could be false if:
```

Reply:
I'll try to remember to fix this when I implement the static graph restore counts